### PR TITLE
feat: edit render resolution through data file

### PR DESCRIPTION
**Pull request by wifimug**

### CODE REVIEW

File: src/launch.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def launch(config):
         bpy.context.scene.unit_settings.system = "METRIC"
         bpy.context.scene.unit_settings.scale_length = 1.0  # Set the scale to 1.0 for metric units
         cropcon = CropController(config, collection)
-        scenerender = SceneRenderer(config["outfile"][0], collection)
+        scenerender = SceneRenderer(config["outfile"][0], collection, config["resolution"])
```
Comment (DanielSwarup) [inline src/launch.py:28]: I think it may be better to just pass `config` as one attribute and then access it later

File: tests/test_data.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+crop:
+  num_rows: 2
+  percentage_share:
```
Comment (DanielSwarup) [inline tests/test_data.yml:3]: Isn't this file auto-generated via the tests? This shouldn't be pushed if it is.

**Review by DanielSwarup — CHANGES_REQUESTED** [review]

**Review by DanielSwarup — APPROVED** [review]
LGTM!

**Comment by DanielSwarup** [comment]
Would recommend "Squash Merge"

**Review by taibohess — APPROVED** [review]

**Pull request merged**